server/public/index.html

```diff
@@ -3,14 +3,27 @@
 <head>
   <meta charset="UTF-8">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Final Project</title>
-  <link rel="icon" href="/favicon.ico">
-  <link rel="stylesheet" href="/reset.css">
-  <link rel="stylesheet" href="/layout.css">
-  <link rel="stylesheet" href="/styles.css">
+  <title>Paws N Pals</title>
+  <link rel="icon" href="favicon.ico">
+  <link rel="stylesheet" href="./reset.css">
+  <link rel="stylesheet" href="./layout.css">
+  <link rel="stylesheet" href="./styles.css">
+  <link rel="stylesheet" href="./override.css">
```

Comment:
Your own CSS should be loaded after the 3rd-party CSS so that you don't run into issues with source order. It will make overriding things much easier to understand.